.git-blame-ignore-revs

```diff
@@ -0,0 +1,2 @@
+// Fix react-scripts warnings
+3715e662fcd70598a39cf3a9fd120b04ee461737
```

Comment:
If you're feeling like a chore at some point, moving tbtc.js and tbtc-dapp to use https://github.com/keep-network/eslint-config-keep + react-app (preferably with our options superseding react-app's where there are conflicts) and hitting them with the prettier hammer is something I've been meaning to do for probably 3-4 months 🙃